ui/src/app/pages/login/account-creation/account-creation-survey.tsx

```diff
@@ -138,70 +140,75 @@ export class AccountCreationSurvey extends React.Component<AccountCreationSurvey
       </ListPageHeader>
 
       {/*Race section*/}
-      <Section header='1. Race'>
+      <Section header='Race'>
         <div style={{display: 'flex', justifyContent: 'flex-start', flexWrap: 'wrap'}}>
           {AccountCreationOptions.race.map((race) => {
-            return this.createOptionCheckbox(race.label, 'race', race.value);
+            return this.createOptionCheckbox(race.label, 'race', race.value, race.value.toString());
```

Comment:
What impact does this change have?

Reply:
It removes a console error message. If you don't provide keys to react within a map, it complains with red in the console.